### ISSUE TITLE
mint button is not showing and it keeps loading

### ISSUE THREAD

**Issue by vishrut66** [FTM-Labs/Aptos-NFT-Mint#14]
![2022-10-21 (2)](https://user-images.githubusercontent.com/108527286/197243552-8bf256cf-c35e-4ee8-a7ed-4975ae9db401.png)

**Comment by aekiratli** [comment]
Open your network tab, resource is probably throwing 404 error. Change `candyMachineInfo.js` accordingly. Note that collection name should be same as the `config.json`

**Comment by vishrut66** [comment]
yeah, it worked. how can i use arweave to upload nfts?

**Comment by erdnc13** [comment]
I am getting the same error, what exactly should I do?

**Comment by vishrut66** [comment]
in candyMachineInfo.js(frontend) file the collection name should be the same as the config.json file.

**Comment by Redmon540** [comment]
yeah, I am getting the same error. what i do?

**Comment by qualityrender** [comment]
Yes I have the same loading issue even that the `config.js` and `candyMachineInfo.js` have the same collection name? really grateful if someone knows what could be the issue

**Comment by qualityrender** [comment]
<img width="1456" alt="Screenshot 2022-10-30 at 00 40 02" src="https://user-images.githubusercontent.com/79476716/198855039-162a8574-dc38-4387-988a-343aeaa52b17.png">
Get this error in consol
